DEVELOPER_GUIDE.md

```diff
@@ -65,16 +67,44 @@ For this reason, every operation *must* be accompanied by the `x-operation-group
 
 ## Grouping Schemas
 
-Schemas are grouped by categories to keep their names short and aid in client generation:
+Schemas are grouped by categories to keep their names short, and aid in client generation (where the schemas are translated into data types/classes, and divided into packages/modules). The schema file names can be in one of the following formats:
 
 - `_common` category holds the common schemas that are used across multiple namespaces and features.
 - `_common.<sub_category>` category holds the common schemas of a specific sub_category. (e.g. `_common.mapping`)
 - `<namespace>._common` category holds the common schemas of a specific namespace. (e.g. `cat._common`, `_core._common`)
 - `<namespace>.<action>` category holds the schemas of a specific sub_category of a namespace. (e.g. `cat.aliases`, `_core.search`)
 
+## Superseded Operations
+
+When an operation is superseded by another operation with **IDENTICAL FUNCTIONALITY**, that is a rename or a change in the URL, it should be listed in [_superseded_operations.yaml](./spec/_superseded_operations.yaml) file. The merger tool will automatically generate the superseded operation in the OpenAPI spec. The superseded operation will have `deprecated: true` and `x-ignorable: true` properties to indicate that it should be ignored by the client generator.
```

Comment:
I don't know if we need to SHOUT :)

Reply:
Because I really really wanted to STRESS that :P
Gonna downgrade it to some **bold** text.